src/molecules/form/index.jsx

```diff
@@ -2,31 +2,44 @@ import { StyledFieldset } from "../../ions/styles";
 import Input from "../../atoms/input";
 import Button from "../../atoms/button";
 import React, { useState } from "react";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import useStore from "../../ions/useStore";
 
 const Form = () => {
-	const [value, setValue] = useState("");
-	const [name, setName] = useState("");
+	const name = useStore(state => state.name);
+	const setName = useStore(state => state.setName);
+	const router = useRouter();
+	const handleSubmit = () => {
+		router.push({
+			pathname: "/lobby/195",
+		});
+	};
+
+	//console.log(router);
+	//return null;
+
 	return (
 		<form
 			onSubmit={event_ => {
 				event_.preventDefault();
-				setName(value);
+				handleSubmit();
 			}}
 		>
+			<h1>{name} Test</h1>
```

Comment:
do you really need Test here?
if you want to have something written there you could try for example:
`{name || "FallbackWhenNameIsNotDefined"}`

Reply:
That was a little test, also removed that one! Thank you :)